.muffet-filter/mfw

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# muffet-filter wrapper
+# ----------------------------------------------------------------------------
+PROJECT_ORG="bhamail"
+PROJECT_NAME="muffet-filter"
+PROJECT_URL="https://github.com/$PROJECT_ORG/$PROJECT_NAME/muffet-filter"
```

Comment:
Is this right? `/muffet-filter` smells funny

Reply:
ex: `https://github.com/bhamail/muffet-filter/muffet-filter` = 404

Reply:
yes, it's right. What's smells funny about it?

Reply:
Just not usually how GitHub URLs are formed, IIRC. 

Happy to push it through if it's been behaving for you.